src/Template/Element/sentences/sentence_form.ctp

```diff
@@ -10,13 +10,13 @@
     </div>
 </div>
 
-<div name="sentenceForm" layout="column" ng-if="!(vm.sentence.audios && vm.sentence.audios.length > 0)">
+<div name="sentenceForm" layout="column" ng-if="!(vm.sentence.audios && vm.sentence.audios.length > 0 && vm.sentence.transcriptions.length === 0)">
 
     <div layout="row" layout-align="start center" ng-if="vm.sentence.permissions.canEdit">
         <md-input-container flex="50">
             <?php /* @translators: language field label on sentence addition form */ ?>
             <label><?= __('Language') ?></label>
-            <md-select ng-model="vm.sentence.lang">
+            <md-select ng-model="vm.sentence.lang" ng-disabled="<?php echo $hasAudio ?>">
```

Comment:
While your code works, it would be more consistent with the rest to use the AngularJS condition `vm.sentence.audios && vm.sentence.audios.length > 0` instead. Then, you would not have to pass on the `$hasAudio` variable.